pom.xml

```diff
@@ -61,6 +61,7 @@
     <module>standalone-metastore</module>
     <module>kafka-handler</module>
     <module>iceberg</module>
+    <module>itests/test-jdbc</module>
```

Comment:
Why we add this module in root pom?

Shouldn't this module be placed here https://github.com/apache/hive/blob/master/itests/pom.xml?

And then build this module through `-Pitests`
`mvn clean package -Pitests`

Reply:
ack, should go into itests/pom.xml